iceoryx2/tests/node_tests.rs

```diff
@@ -248,7 +281,7 @@ mod node {
         let number_of_creators = (SystemInfo::NumberOfCpuCores.value()).clamp(2, 1024);
         const NUMBER_OF_ITERATIONS: usize = 100;
         let barrier = Barrier::new(number_of_creators);
-        let mut config = Config::global_config().clone();
+        let mut config = generate_isolated_config();
```

Comment:
@elfenpiff  for some reasons, the `concurrent_node_creation_and_listing_works` still fails. I tried to debug but could not find the root cause.